images/jre/configs/openjdk-11.apko.yaml

```diff
@@ -19,14 +20,14 @@ accounts:
       uid: 65532
   run-as: java
 
-work-dir: /app
+work-dir: /home/build
```

Comment:
could this be breaking?

Reply:
potentially yeah, if folks are using absolute paths within their build environment. do we have precedence for these kind of changes?

Reply:
I'll defer to @imjasonh on making a call on this.

Reply:
maybe move to a different PR or is this related to the jdk refactoring?

Reply:
Not related.  `/home/build` is just the default build workspace in melange, but does not matter here, except for scenarios where we might want it to be consistent for education reasons.